src/main/java/com/craftingguide/exporter/extensions/craftingguide/ItemIconDumperScreen.java

```diff
@@ -370,7 +376,7 @@ private void exportAllItems() {
 
     private boolean isExporting = false;
 
-    private LinkedList<ItemModel> items = new LinkedList<>();
+    private ArrayList<ItemModel> items = new ArrayList<>();
```

Comment:
Let's not change this to an `ArrayList`.  I specifically chose a `LinkedList` because I knew I'd be pulling items off the front of it one-by-one.  With an `ArrayList`, you need to re-shuffle all the rest of the items each time you do that.  With a `LinkedList`, you don't.

Reply:
changing this back works fine